client/blocks/jetpack-backup-creds-banner/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+
+class JetpackBackupCredsBanner extends Component {
+	static propTypes = {
+		// Tracks event name on click
+		event: PropTypes.string.isRequired,
+		// Connected props
+		siteId: PropTypes.number,
```

Comment:
We seem to be missing `isJetpack` here.

Also, maybe it would make sense to add `translate` under another comment saying `From localize() HoC` or something.